web-services/client/src/main/java/datawave/webservice/model/Model.java

```diff
@@ -133,28 +138,30 @@ public String getMainContent() {
         builder.append("<div id=\"myTable_wrapper\" class=\"dataTables_wrapper no-footer\">\n");
         builder.append("<table id=\"myTable\" class=\"dataTable no-footer\" role=\"grid\" aria-describedby=\"myTable_info\">\n");
 
-        builder.append("<thead><tr><th>Visibility</th><th>FieldName</th><th>DataType</th><th>ModelFieldName</th><th>Direction</th><th>Lenient</th></tr></thead>");
+        builder.append("<thead><tr><th>Visibility</th><th>FieldName</th><th>DataType</th><th>ModelFieldName</th><th>Direction</th><th>Attributes</th></tr></thead>");
         builder.append("<tbody>");
 
-        // first gather the model fields that are deemed "lenient" (i.e. where the model field name has a mapping to the field called "lenient")
-        Set<String> lenientModelFields = new HashSet<>();
+        // first gather the model fields that are deemed "lenient" (i.e. where the model field name has a lenient attribute)
+        Multimap<String,String> modelFieldAttributes = HashMultimap.create();
         for (FieldMapping field : this.getFields()) {
-            if (field.getDirection() == Direction.FORWARD) {
-                if (field.isLenientMarker()) {
-                    lenientModelFields.add(field.getModelFieldName());
-                }
+            if (!field.isFieldMapping()) {
+                modelFieldAttributes.putAll(field.getModelFieldName(), field.getAttributes());
             }
         }
 
         for (FieldMapping f : this.getFields()) {
             // don't include the lenient marker itself
```

Comment:
should this be updated?
```suggestion
            // don't include model field attributes
```